test/StringUtility.test.cpp

```diff
@@ -4,6 +4,15 @@
 
 using namespace StringUtility;
 
+class ImplicitStringConversionTestFixture {
+public:
+	std::string data;
+
+	operator std::string() const {
+		return data;
+	}
```

Comment:
We can try adding `explicit` to this method to check the previous suggestion.

```cpp
explicit operator std::string() const
```

----

https://en.cppreference.com/w/cpp/language/explicit